src/ColorPicker/widget/ColorPicker.js

```diff
@@ -32,80 +26,88 @@ mxui.widget.declare('ColorPicker.widget.ColorPicker', {
     mxobj           : null,
     _hasStarted     : false,
     ignoreChange    : false,
-
-    startup : function(){
+    
+    postCreate : function(){
+        dom.addCss('widgets/ColorPicker/widget/lib/spectrum.css');
+        dom.addCss('widgets/ColorPicker/widget/ui/ColorPicker.css');
```

Comment:
No need of adding this line. As long as the CSS file in the correct folder and path, it will be picked.